coral/src/domain/topic/topic-transformer.ts

```diff
@@ -20,4 +23,71 @@ function transformTopicApiResponse(
+const ADVANCED_TOPIC_CONFIG_DOCUMENTATION: Record<string, string> = {
+  "cleanup.policy":
+    "https://kafka.apache.org/documentation/#topicconfigs_cleanup.policy",
+  "compression.type":
+    "https://kafka.apache.org/documentation/#topicconfigs_compression.type",
+  "delete.retention.ms":
+    "https://kafka.apache.org/documentation/#topicconfigs_delete.retention.ms",
+  "file.delete.delay.ms":
+    "https://kafka.apache.org/documentation/#topicconfigs_file.delete.delay.ms",
+  "flush.messages":
+    "https://kafka.apache.org/documentation/#topicconfigs_flush.messages",
+  "flush.ms": "https://kafka.apache.org/documentation/#topicconfigs_flush.ms",
+  "follower.replication.throttled.replicas":
+    "https://kafka.apache.org/documentation/#topicconfigs_follower.replication.throttled.replicas",
+  "index.interval.bytes":
+    "https://kafka.apache.org/documentation/#topicconfigs_index.interval.bytes",
+  "leader.replication.throttled.replicas":
+    "https://kafka.apache.org/documentation/#topicconfigs_leader.replication.throttled.replicas",
+  "max.compaction.lag.ms":
+    "https://kafka.apache.org/documentation/#topicconfigs_max.compaction.lag.ms",
+  "max.message.bytes":
+    "https://kafka.apache.org/documentation/#topicconfigs_max.message.bytes",
+  "message.downconversion.enable":
+    "https://kafka.apache.org/documentation/#topicconfigs_message.downconversion.enable",
+  "message.format.version":
+    "https://kafka.apache.org/documentation/#topicconfigs_message.format.version",
+  "message.timestamp.difference.max.ms":
+    "https://kafka.apache.org/documentation/#topicconfigs_message.timestamp.difference.max.ms",
+  "message.timestamp.type":
+    "https://kafka.apache.org/documentation/#topicconfigs_message.timestamp.type",
+  "min.cleanable.dirty.ratio":
+    "https://kafka.apache.org/documentation/#topicconfigs_min.cleanable.dirty.ratio",
+  "min.compaction.lag.ms":
+    "https://kafka.apache.org/documentation/#topicconfigs_min.compaction.lag.ms",
+  "min.insync.replicas":
+    "https://kafka.apache.org/documentation/#topicconfigs_min.insync.replicas",
+  preallocate:
+    "https://kafka.apache.org/documentation/#topicconfigs_preallocate",
+  "retention.bytes":
+    "https://kafka.apache.org/documentation/#topicconfigs_retention.bytes",
+  "retention.ms":
+    "https://kafka.apache.org/documentation/#topicconfigs_retention.ms",
+  "segment.bytes":
+    "https://kafka.apache.org/documentation/#topicconfigs_segment.bytes",
+  "segment.index.bytes":
+    "https://kafka.apache.org/documentation/#topicconfigs_segment.index.bytes",
+  "segment.jitter.ms":
+    "https://kafka.apache.org/documentation/#topicconfigs_segment.jitter.ms",
+  "segment.ms":
+    "https://kafka.apache.org/documentation/#topicconfigs_segment.ms",
+  "unclean.leader.election.enable":
+    "https://kafka.apache.org/documentation/#topicconfigs_unclean.leader.election.enable",
+};
+
+function transformGetTopicAdvanvedConfigOptionsResponse(
```

Comment:
Although the function is not doing a lot, I think it's still worth adding a  test for it!